File: tmp/nx-e2e/proj/apps/boot-maven-app-9727701/src/main/java/com/example/bootmavenapp9727701/HelloController.java
package com.example.bootmavenapp9727701;

import com.example.bootmavenlib3433640.HelloService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

  @Autowired
  private HelloService helloService;

  @GetMapping("/")
  public String greeting() {
    return this.helloService.message();
  }
}

File: tmp/nx-e2e/proj/apps/boot-maven-app-4532651/src/main/kotlin/com/example/bootmavenapp4532651/HelloController.kt
package com.example.bootmavenapp4532651
import com.example.bootmavenlib495386.HelloService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController(@Autowired val helloService: HelloService) {

  @GetMapping("/")
  fun greeting(): String = helloService.message()
}
